content/en/docs/reference/access-authn-authz/extensible-admission-controllers.md

```diff
@@ -736,7 +736,55 @@ webhook to be called.
 
 Here is an example illustrating a few different uses for match conditions:
 
-{{< codenew file="access/admission-webhook-match-conditions.yaml" >}}
+```yaml
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+webhooks:
+  - name: my-webhook.example.com
+    matchPolicy: Equivalent
+    rules:
+      - operations: ['CREATE','UPDATE']
+        apiGroups: ['*']
+        apiVersions: ['*']
+        resources: ['*']
+    failurePolicy: 'Ignore' # Fail-open (optional)
+    sideEffects: None
+    clientConfig:
+      service:
+        namespace: my-namespace
+        name: my-webhook
+      caBundle: '<omitted>'
```

Comment:
To reviewers: this one is not READY for users to try out, so it should be used as an inline example.

Reply:
Not ready because the CA bundle is missing? I think that's the only piece, but I might have missed something.

Reply:
Yes. That string fails the validation. It also implies that this one is not for users to paste and run.